source/core/options.ts

```diff
@@ -3,8 +3,8 @@ import {Buffer} from 'node:buffer';
 import {promisify, inspect} from 'node:util';
 import {URL, URLSearchParams} from 'node:url';
 import {checkServerIdentity} from 'node:tls';
-import {request as httpRequest} from 'node:http';
-import {request as httpsRequest} from 'node:https';
+import http from 'node:http';
+import https from 'node:https';
```

Comment:
Add a short comment about it, otherwise this could accidentally be regressed in the future.